src/ast/passes/macro_expansion.cpp

```diff
@@ -171,7 +170,7 @@ void MacroExpander::visit(Expression &expr)
 
 std::string MacroExpander::get_new_var_ident(std::string original_ident)
 {
-  return std::format("$${}_{}", macro_name_, original_ident);
+  return std::string("$$") + macro_name_ + std::string("_") + original_ident;
```

Comment:
Not going to make you re-submit this PR but in the future you can just do `return "$$" + macro_name_ + "_" + original_ident;`

Reply:
Thanks!